### PR TITLE
Feature/testing api ocr

### PR DESCRIPTION
**Pull request by joshua-araujo**
## Description
Created script to test functions of Box API to retrieve information from folders and download files.  Once files are downloaded we run OCR on them.  Dependencies required for this change is boxsdk.

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [ ] Test B

**Test Configuration**:
* SDK: boxsdk

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: app/testing.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import os
+import re
+from pdf2image import convert_from_path, convert_from_bytes
+import pytesseract
```
Comment (pflee1989) [inline app/testing.py:4]: Great work, let's build this out to automate more retrieval transactions.

**Review by pflee1989 — APPROVED** [review]

**Review by andrewsbusby — APPROVED** [review]
Looks good to me as far as I can tell.

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This is looking pretty good, but I'd like for you to include the standard Loom recording explaining this PR and showcasing the changes you made - requesting changes until that recording is included!

File: app/testing.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+client = Client(auth)
+
+# # Download file from Box
+file_id = 844529989327
+file_content = client.file(file_id).content()
+
+# Set user
+me = client.user().get()
```
Comment (tinomen) [inline app/testing.py:24]: Why are we setting this unused variable?

**Pull request closed without merging**